clippy_lints/src/methods.rs

```diff
@@ -342,19 +342,19 @@ declare_clippy_lint! {
 ///
 /// **Example:**
 /// ```rust
-/// foo.expect(&format("Err {}: {}", err_code, err_msg))
+/// foo.expect(&format!("Err {}: {}", err_code, err_msg))
 /// ```
 /// or
 /// ```rust
-/// foo.expect(format("Err {}: {}", err_code, err_msg).as_str())
+/// foo.expect(format!("Err {}: {}", err_code, err_msg).as_str())
 /// ```
 /// this can instead be written:
 /// ```rust
 /// foo.unwrap_or_else(|_| panic!("Err {}: {}", err_code, err_msg))
 /// ```
 /// or
 /// ```rust
-/// foo.unwrap_or_else(|_| panic!(format("Err {}: {}", err_code, err_msg).as_str()))
+/// foo.unwrap_or_else(|_| panic!(format!("Err {}: {}", err_code, err_msg).as_str()))
```

Comment:
This suggestion doesn't compile. [Playground Link](https://play.rust-lang.org/?gist=403688525d8e0c6b930528ffd9629d38&version=stable&mode=debug&edition=2015)
This is actually an instance of the bug in #2937. 

I guess you can just delete it since Clippy should suggest the other form anyway.

Reply:
I feel sorry for creating pull request without test.

I will update this PR, right now.